content/posts/0004-writing-buildpacks.md

```diff
@@ -0,0 +1,97 @@
+
+
+## Creating a Paketo-style buildpack
+
+There are definitely some need-to-know basics when it comes to buildpacks that have been well documented by the Cloud Native Buildpacks project in [the docs](https://buildpacks.io/docs/concepts/). Major concepts such as the buildpack lifecycle phases, and the other components that work alongside buildpacks are explained there. Once you have the basics down, there is a [simple tutorial](https://buildpacks.io/docs/buildpack-author-guide/create-buildpack/) to create a buildpack in Bash.
+
+
+### Packit is here to help!
+
+Kicking things up a notch, the [Creating a Paketo Buildpack tutorial](https://paketo.io/docs/tutorials/create-paketo-buildpack/) on the [Paketo website](paketo.io) outlines how to create a really simple buildpack in **Golang**. With Golang, it’s possible to implement more complex logic with less work than with Bash. A shining star of the tutorial is the introduction of the [Packit library](github.com/paketo-buildpacks/packit), which simplifies the repetitive parts of writing a buildpack, such as [dependency installation](https://github.com/paketo-buildpacks/packit/blob/main/postal) and [file system manipulation](https://github.com/paketo-buildpacks/packit/blob/main/fs). As a part of the Cloud Native Buildpacks project, our buildpacks follow the [CNB Spec](https://github.com/buildpacks/spec). A huge benefit of Packit is that it helps abide by the CNB spec without too much effort.
+
+Writing a buildpack with the Packit library is integral to how we write our buildpacks and keep them consistent. I recommend checking out Packit, and then also looking at one of our buildpacks to see how the packages are used. Key an eye out for future blog post deep diving into Packit as well.
+
+
+### Bootstrap it
+
+Another huge help is our buildpack [Bootstrapper](https://github.com/paketo-community/bootstrapper) which sets up a basic buildpack directory structure in a Packit-compliant way. Using the bootstrapper tool removes the guesswork involved in setting up all the basic pieces of a buildpack, and enforces test-driven development. The bootstrapper-provided integration tests fail by default, encouraging you to write tests first and think about the overarching picture of what you need your buildpack to do.
```

Comment:
```suggestion
Another huge help is our buildpack [Bootstrapper](https://github.com/paketo-community/bootstrapper) which sets up a basic buildpack directory structure in a Packit-compliant way. Using the bootstrapper tool removes the guesswork involved in setting up all the basic pieces of a buildpack, and lets you test the buildpack along the way. The bootstrapper-provided integration tests fail by default, encouraging you to write tests first and think about the overarching picture of what you need your buildpack to do.
```
"Enforces" feels a bit strong here

Reply:
I want to mention test-driven development still. How about:
"and allows you the option of test driven development"